tools/stern/stern.integration.yaml

```diff
@@ -0,0 +1,26 @@
+stern:
```

Comment:
Do we need this file? There are no integration tests for stern.

Reply:
No - we don't; thanks for pointing it out.